ZenPacks/zenoss/Import4/model.py

```diff
@@ -188,7 +197,12 @@ def _check_files(self):
 
         # check egg directories
         self.zenpack_count = int(subprocess.check_output(
```

Comment:
You should probably check the output code here first before arbitrarily taking the output